src/structures/MusicHandler.js

```diff
@@ -103,8 +107,7 @@ module.exports = class MusicHandler {
             this.queue.unshift(this.queue[to - 1]);
             this.queue.splice(to, 1);
         } 
-       if (this.player.paused) await this.player.pause(false);
```

Comment:
i removed this because it's unnecessary, the player will automatically resume when new track is playing